cfpb/cfgov-refresh
# Create hmda app for new cf.gov/hmda pages

Early stage WIP

## Additions

- `cfgov/hmda` app
- `HmdaExplorerPage` page type that will replace [hmda-explorer](https://www.consumerfinance.gov/hmda/explore)

## Removals

-

## Changes

-

## Testing

1. Import a recent production db dump: `./refresh_data.sh`
2. Head over to the new [HMDA sublanding page](http://0.0.0.0:8000/admin/pages/12493/).
3. Create a `HMDA Explorer` child page and add any page content that you'd like.
5. Publish the page and view it. You should see the data filter controls and a results list (along with any content you added to the page).

## Screenshots


## Notes

-

## Todos

-

## Checklist


## Testing checklist

### Browsers


### Accessibility


### Other
